### PR TITLE
I3s picking refactoring

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: examples/experimental/i3s-picking/app.js [hunk 1]
```diff
@@ -196,19 +196,43 @@ export default class App extends PureComponent {
...
       ? JSON.stringify(attributes, null, 2).replace(/[\{\}']+/g, '')
       : 'loading metadata...';
   }
 
   handleClosePanel() {
-    this.setState({selectedAttribute: null});
+    this.setState({selectedFeatureAttributes: null});
+  }
+
+  getSelectedFeatureAttributes(tile, featureIndex) {
```
Comment (ibgreen) [inline examples/experimental/i3s-picking/app.js:209]: Good improvement, but ultimately this should be a function in the tiles module, not the app. It does not depend on any UI or deck.gl knowledge.

File: modules/i3s/src/i3s-loader.js [hunk 2]
```diff
@@ -31,7 +31,8 @@ export const I3SLoader = {
       isTileHeader: 'auto',
       tile: null,
       tileset: null,
-      useDracoGeometry: true
+      useDracoGeometry: true,
+      loadFeatureAttributes: true
```
Comment (ibgreen) [inline modules/i3s/src/i3s-loader.js:35]: Don't forget to document.

File: modules/i3s/test/i3s-loader.spec.js [hunk 3]
```diff
@@ -46,3 +46,11 @@ test('I3SLoader#DRACO geometry', async t => {
 
   t.end();
 });
+
+test('I3SLoader parsing featureAttributes', async t => {
```
Comment (ibgreen) [inline modules/i3s/test/i3s-loader.spec.js:50]: test case with flag set to true/false?

**Review by ibgreen — APPROVED** [review]

File: examples/experimental/i3s-picking/app.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-picking/app.js]: Done

File: modules/i3s/src/i3s-loader.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply modules/i3s/src/i3s-loader.js:35]: Updated docs

File: modules/i3s/test/i3s-loader.spec.js (hunk 3 above)
Comment (dryabinin-actionengine) [reply modules/i3s/test/i3s-loader.spec.js]: Added more tests

**Pull request merged**